source/initial-dns-seedlist-discovery/tests/one-txt-record.json

```diff
@@ -1,5 +1,5 @@
 {
-  "uri": "mongodb+srv://test5.test.build.10gen.cc/?replicaSet=repl0",
+  "uri": "mongodb+srv://test5.test.build.10gen.cc/",
```

Comment:
It would be helpful if any test that asserts the value of authSource also included user:pwd in the connection string, as some drivers only provide access to the authSource via a credential object (which requires a username)

Reply:
Added.